README.md

```diff
@@ -5,8 +5,27 @@
 
 ### Getting the data and setting it up
 
-Download `final_train.txt` from Google Drive at https://drive.google.com/open?id=1HEv4T1ler395YVoJi6_CMk1_9gg5Ua2Z/ and extract it to the directory `data`.
+1. In the repository root, create a directory `data`.
 
+2. Download `train_conll.txt` from Google Drive at https://drive.google.com/file/d/1fzhNn_baYWE6Ltxf1VKlkYzCUwUlHVXx/view?usp=sharing/ and extract it to `data`.
+
+3. Downlaod `trial_conll.txt` from Google Drive at https://drive.google.com/file/d/1T_AvUYwc8Fld_8VoUj9s9S1ZbmpkAsVI/view?usp=sharing/ and extract it to `data`.
+
+4. Inside `data`, create a directory `SST-3`.
+
+5. Inside `SST-3`, create a directory `checks`.
+
+6. From root, run `python3 clean_data.py`.
+
+7. (If you wish to append SST-2 data) From root, run `python3 append_glue.py`.
+
+### Run GLUE experiments
+
+From root, run `sh glue.sh`.
+
+### Results
```

Comment:
Add instructions about running baseline.

Reply:
@mhrmm I have added a few instructions to the README:

1. How to run the baseline experiment and where to find the results.
2. Where to find the validation training and dev sets for the GLUE exps.
3. Where to find the results of run_glue.py in the output_dir.

I haven't done a simple accuracy on the baseline yet, but I will add that. I thought it might be more useful for you to look at the tensors themselves in the output right now. The baseline results are in the form `index    output_tensor    predicted_label`.